python/lsst/resources/s3utils.py

```diff
@@ -147,6 +148,10 @@ def getS3Client() -> boto3.client:
     -----
     The endpoint URL is from the environment variable S3_ENDPOINT_URL.
     If none is specified, the default AWS one is used.
+
+    If the environment variable LSST_CEPH_BUCKETS exists (with any value),
```

Comment:
I'm concerned that CEPH is not the only reason that someone would want to do this. How about calling it something like `LSST_S3_DISABLE_VALIDATION` ? Also, I'm not sure I like it that we don't have to set it to a truthy value.